### DISCUSSION TITLE
How to Change Account Email

### DISCUSSION THREAD

**Discussion by Isuru-Nanayakkara** [codemagic-ci-cd/codemagic-docs#1096] [Q&A]
Hello, is there a way to change a Codemagic account's login email?

Failing that, can I delete an account?

**Comment by K-Codemagic** [comment]
Hi, you can create a new Codemagic account, however, if you have running apps in Codemagic you can make the changes by creating a TEAM. 

Perform the following steps:

1. Create a team, if there isn't
2. Add the apps that are necessary
3. Invite the new user's email.
4. Make them as owner of the team.
5. Downgrade the role of the old email user in the team.
6. Remove the old user.
7. Then you can remove the apps from shared applications in the team, which will move the apps to the new users' account, and then delete the team(If not required)

Note: Step 7 is only applicable if you don't want teams!
